Core/include/Acts/TrackFitting/KalmanFitter.hpp

```diff
@@ -325,7 +325,10 @@ class KalmanFitter {
         return;
       }
 
-      ACTS_VERBOSE("KalmanFitter step");
+      ACTS_VERBOSE("KalmanFitter step at pos: "
+                   << stepper.position(state.stepping).transpose()
```

Comment:
thanks for transposing them. this saves a lot of log lines

Reply:
I wonder if anyone ever needed the non-transposed print-out of an `Eigen::Vector` :D